.examples/README.md

```diff
@@ -13,7 +13,7 @@ The Dockerfiles use the default images as base image and build on top of it.
 
 Example | Description
 ------- | -------
-[cron](https://github.com/nextcloud/docker/tree/master/.examples/dockerfiles/cron) | uses supervisor to run the cron job inside the container (so no extra container is needed).
+[cron](https://github.com/nextcloud/docker/tree/master/.examples/dockerfiles/cron) | uses supervisor to run the cron job inside the container (so no extra container is needed). This image runs `supervisord`. To enable auto install and update run images like this with an additional environment variable:  `NEXTCLOUD_UPDATE=1`. See [here](https://github.com/nextcloud/docker/blob/master/README.md#auto-configuration-via-environment-variables) for more information.
```

Comment:
It would also make sense to set it for the user in the Dockerfile

Reply:
Can it then be overwritten by explicitly setting another value?

Reply:
I am not sure. I would only add it to the supervisor example. And if a user doesn't want it he can simply comment it out before building his custom image.

Reply:
And the supervisor example is basically useless without the updater working.